### PR TITLE
ProperCase  to snake_case

### PR DESCRIPTION
**Pull request by bwockner**

### CODE REVIEW

File: src/experimental/SMI_DrillCoreProject.py [hunk 1]
```diff
@@ -145,19 +146,21 @@ def make_collar_map(self):
...
     context = udSDK.udContext()
-    try:
-        context.try_resume("https://udstream.euclideon.com", "pythonProjectDownloader", argv[1])
-    except udSDK.UdException:
-        context.connect_legacy("https://udstream.euclideon.com", "pythonProjectDownloader", argv[1], argv[2])
+
+    #try:
+        #context.try_resume("https://udstream.euclideon.com", "pythonProjectDownloader", argv[1])
+    #except udSDK.UdException:
+        #context.connect_legacy("https://udstream.euclideon.com", "pythonProjectDownloader", argv[1], argv[2])
```
Comment (OsiJr) [inline src/experimental/SMI_DrillCoreProject.py:153]: This seems unrelated?

**Review by samsinsane — APPROVED** [review]
